scripts/ci/Dockerfile

```diff
@@ -105,125 +110,43 @@ USER runner
 
 CMD [ "jest", "main.spec.js" ]
 
-FROM cypress/base:14.17.6 as cypress-base
-
-USER root
-RUN id
-
-RUN node --version
-
-# Chrome dependencies
-RUN apt-get update && \
-  apt-get install -y \
-  fonts-liberation \
-  libcurl4 \
-  libcurl3-gnutls \
-  libcurl3-nss \
-  xdg-utils \
-  wget \
-  curl \
-  # clean up
-  && rm -rf /var/lib/apt/lists/* \
-  && apt-get clean
-
-# install libappindicator3-1 - not included with Debian 11
-RUN wget --no-verbose /usr/src/libappindicator3-1_0.4.92-7_amd64.deb "http://ftp.us.debian.org/debian/pool/main/liba/libappindicator/libappindicator3-1_0.4.92-7_amd64.deb" && \
-  dpkg -i /usr/src/libappindicator3-1_0.4.92-7_amd64.deb ; \
-  apt-get install -f -y && \
-  rm -f /usr/src/libappindicator3-1_0.4.92-7_amd64.deb
-
-# install Chrome browser
-RUN wget --no-verbose -O /usr/src/google-chrome-stable_current_amd64.deb "http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_100.0.4896.88-1_amd64.deb" && \
-  dpkg -i /usr/src/google-chrome-stable_current_amd64.deb ; \
-  apt-get install -f -y && \
-  rm -f /usr/src/google-chrome-stable_current_amd64.deb
-
-# "fake" dbus address to prevent errors
-# https://github.com/SeleniumHQ/docker-selenium/issues/87
+FROM mcr.microsoft.com/playwright:v1.25.2-focal as cypress-base
```

Comment:
Might be confusing to have this step be named cypress-base still

Reply:
👍🏻